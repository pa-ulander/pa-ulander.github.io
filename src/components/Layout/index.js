import React from 'react'
import HeaderNav from '../HeaderNav'

const Layout = ({ children }) => {
  return (
    <>
      <HeaderNav />
      <div className='container'>{children}</div>
    </>
  )
}

export default Layout
