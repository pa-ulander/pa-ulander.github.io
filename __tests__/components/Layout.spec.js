import React from 'react'
import { shallow } from 'enzyme'
import Layout from '../../src/components/Layout'

describe('Layout component', () => {
  it('should render correctly', () => {
    const component = shallow(<Layout />)
    expect(component).toMatchSnapshot()
  })
})
