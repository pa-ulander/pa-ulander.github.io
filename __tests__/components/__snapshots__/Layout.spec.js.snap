// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component should render correctly 1`] = `
<Fragment>
  <HeaderNav />
  <div
    className="container"
  />
</Fragment>
`;
